release.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+import argparse
+import os
+import subprocess
+
+from build_and_push_images import IMAGES, docker_build, docker_push
+
+
+def ask_yes_no(question):
+    if input(question).lower().strip()[0] == "y":
+        return True
+    return False
+
+
+def continue_or_stop(text):
+    print(text)
+    if not ask_yes_no("are you sure to continue? (y/n) "):
+        print("exiting.")
+        exit(0)
+
+
+def execute(command, shell=False):
+    if isinstance(command, str):
+        command = command.split()
+    subprocess.check_call(command, shell=shell)
+
+
+def execute_steps(steps):
+    for step in steps:
+        execute(step)
+
+
+def build_and_push_images(release_version):
+    for image in IMAGES:
+        docker_build(release=release_version, **image)
+    for image in IMAGES:
+        docker_push(release=release_version, **image)
+
+
+def prepare_for_release(release_version):
+    execute_steps([
+        "git checkout develop",
+        "git pull --ff-only",
+        "git flow release start {}".format(release_version),
+    ])
+
+
+def make_release_specific_changes(release_version):
+    execute_steps([
+        "python ./web_frontend/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
+        "python ./osmaxx_conversion_service/manage.py makemessages -l de_CH -l en -l en_UK -l en_US",
+        ["git", "commit", "-m", 'added makemessages output', 'osmaxx/locale'],
+    ])
+    version_file_path = os.path.join(os.path.dirname(__file__), 'osmaxx', '__init__.py')
+    with open(version_file_path, 'r') as version_file:
+        version_file_content = version_file.readlines()
+    with open(version_file_path, 'w') as version_file:
+        for line in version_file_content:
+            if line.startswith('__version__'):
+                line = "__version__ = '{}'".format(release_version)
+            version_file.write(line)
```

Comment:
I didn't know until now, but apparently [`readlines` is considered silly](http://stupidpythonideas.blogspot.ch/2013/06/readlines-considered-silly.html?_escaped_fragment_#!). Probably doesn't matter for a file this small, but you might want to [use `fileinput.input(..., inplace=True)`](http://stackoverflow.com/questions/39086/search-and-replace-a-line-in-a-file-in-python/290494#290494).